docs/source/markdown/podman-system-connection-list.1.md

```diff
@@ -4,13 +4,28 @@
 podman\-system\-connection\-list - List the destination for the Podman service(s)
 
 ## SYNOPSIS
-**podman system connection list**
+**podman system connection list** [*options*]
 
-**podman system connection ls**
+**podman system connection ls** [*options*]
 
 ## DESCRIPTION
 List ssh destination(s) for podman service(s).
 
+## OPTIONS
+
+#### **--format**=*format*
+
+Change the default output format.  This can be of a supported type like 'json' or a Go template.
+Valid placeholders for the Go template listed below:
+
+| **Placeholder** | **Description**                                                               |
+| --------------- | ----------------------------------------------------------------------------- |
+| .Name           | Connection Name/Identifier |
```

Comment:
I think per the new mapage [syntax](https://raw.githubusercontent.com/containers/podman/master/docs/MANPAGE_SYNTAX.md)  these should have asterisks around each place holder ala:

```suggestion
| *.Name*           | Connection Name/Identifier |
```